effekt/shared/src/main/scala/effekt/generator/llvm/Tree.scala

```diff
@@ -17,6 +17,7 @@ export Definition.*
 enum CallingConvention {
   case Ccc()
   case Tailcc()
+  case TailccPerhaps()
```

Comment:
Does anyone have a better idea for calling convention transition from CCC to Tailcc? Here, `TailccPerhaps` is unenforced (not `musttail`) Tailcc, such that it can be used in the initial jump to `main` from the CCC entry.

Reply:
Do I understand correctly that this is ONLY used for the main entry point?

Reply:
Yes, exactly. I don't have ideas for other solutions though.

Reply:
One could also refactor this to 

```
case Tailcc(musttail: Boolean)
```

which is isomorphic, but documents the semantics a bit better than `TailccPerhaps`.